var/spack/repos/builtin/packages/pmdk/package.py

```diff
@@ -18,9 +18,10 @@ class Pmdk(Package):
     git      = "https://github.com/pmem/pmdk.git"
 
     version('develop', branch='master')
+    version('1.6',     sha256='3b99e6c30709326a94d2e73a9247a8dfb58d0a394c5b7714e5c3d8a3ad2e2e9f')
     version('1.5',     sha256='6b069d7207febeb62440e89245e8b18fcdf40b6170d2ec2ef33c252ed16db2d4')
 
-    depends_on('ncurses', when='@1.5:')
+    depends_on('ncurses', when='@1.5')
```

Comment:
Can you put a pointer to some reference that reports version 1.6 not depending on `ncurses`, while previous versions did?

Reply:
Thanks @alalazo , actually after closer inspection ncurses wasn't required for 1.5 either.  I've deleted that line entirely in this PR now.